### DISCUSSION TITLE
Template resolver

### DISCUSSION THREAD

**Discussion by lukeplausin** [cloudreach/sceptre#734] [Ideas]
Hi Sceptre contributors

This isn't an issue as such, but couldn't find any message board or channel to post on.
I've created a sceptre `template` resolver, which may be useful. You can use it to template inputs to files, using any of Sceptre's built in template engines. I'm using it primarily for templating EC2 userdata with Jinja2, but there may well be other uses for it.

https://github.com/lukeplausin/sceptre-template-resolver

Let me know what you think about it!
PS: Feature request - sceptre users forum? 😉 

Luke

**Comment by ngfgrant** [comment]
Hey Luke - awesome thanks. We'll soon move github org to github.com/sceptre and this is exactly the sort of thing we could share there too. I have a couple of custom hook/resolver template repos that can be used to provide ci/cd and pypi deployment too.

Once we move org it would be awesome to pull this in as its own repo there so others can use/contribute?

**Comment by ngfgrant** [comment]
Hey @lukeplausin 

Do you want to give using the new template resolver repo a go? https://github.com/Sceptre/sceptre-resolver-template. It would be good to use your resolver as a first test case!

Would be interesting to see if the docs make any sense: https://github.com/Sceptre/sceptre-resolver-template/wiki
